samples/snippets/src/test/java/com/example/speech/SpeechProfanityFilterTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class SpeechProfanityFilterTest {
+  private static final String AUDIO_FILE = "gs://cloud-samples-tests/speech/brooklyn.flac";
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
```

Comment:
We need to set the capture the original output stream and restore it.

Reply:
Many of the examples in java-docs-samples are incorrect (https://github.com/GoogleCloudPlatform/java-docs-samples/issues/4423).

In the before, you should capture the value of `System.out` and restore it in the `tearDown`

Reply:
Or use the junit rule like: https://github.com/googleapis/java-accessapproval/pull/281/files#diff-2d117883728d4cbda30f1cf369baad5ae5d61abd5f79664624cfc156f05b24ecR32

Reply:
This has now been handled.